src/oaipmh/add_handles.py

```diff
@@ -62,10 +62,17 @@ def create_handles(reader: DictReader, config: dict) -> list:
     exports = []
     for row in reader:
         if 'Handle' not in row or row['Handle'] is None:
+            if row['URI'].startswith(config['BASE_URL']) is False:
```

Comment:
This would be better written as:
```python
if not row['URI'].startswith(config['BASE_URL']):
```
Reference: https://docs.quantifiedcode.com/python-anti-patterns/readability/comparison_to_true.html#compare-values-to-true-using-the-pattern-if-cond-is-true